crowdsourcing/viewsets/task.py

```diff
@@ -72,10 +71,19 @@ def retrieve_with_data(self, request, *args, **kwargs):
         worker_timestamp = task_worker.created_timestamp
         now = datetime.datetime.utcnow().replace(tzinfo=utc)
         time_left = int((timeout * 60) - (now - worker_timestamp).total_seconds())
+
+        auto_accept = False
+        try:
+            user_prefs = UserPreferences.objects.get(user=request.user)
```

Comment:
either use something like request.user.preferences.auto_accept or use get_model_or_none which wraps this for you